Source/WebCore/html/ImageBitmap.cpp

```diff
@@ -175,6 +174,15 @@ std::optional<DetachedImageBitmap> ImageBitmap::detach()
     return DetachedImageBitmap { makeUniqueRefFromNonNullUniquePtr(WTFMove(serializedBitmap)), originClean(), premultiplyAlpha(), forciblyPremultiplyAlpha() };
 }
 
+#if USE(SKIA)
+void ImageBitmap::willTransferBetweenThreads()
```

Comment:
I would use a name to make more explicit that this might change the internal buffer, maybe `prepareForCrossThreadTransfer()` or something like that.